rpmconf/Install/zmsetup.pl

```diff
@@ -1234,6 +1234,11 @@ sub setLdapDefaults {
       if ($config{zimbraVersionCheckNotificationEmailFrom} eq "");
   }
 
+  $config{EphemeralBackendURL} = getLdapConfigValue("zimbraEphemeralBackendURL");
+  if ($config{EphemeralBackendURL} ne "") {
```

Comment:
nit: to be more consistent with syntax in the surrounding area, I'd probably change this line/block to:
```
$config{USEEPHEMERALSTORE} = "yes" if ($config{EphemeralBackendURL} ne "");
```

Reply:
agree with @plobbes